loggins.sh

```diff
@@ -87,7 +91,28 @@ function explain ()
 	fi
 }	
 
+# The ZSHEXIT functions
+function shutdown()
+{
+	echo "Shutting down..."
+	derby stop &> /dev/null
+	set -m
+}
+
+# OTHER USEFULE FUNCTIONS
+function quit()
+{
+	echo "Shutting down..."
+	derby stop &> /dev/null
+	set -m
+	add-zsh-hook -D precmd status
+	add-zsh-hook -D preexec loggins
+	add-zsh-hook -D zshexit shutdown
+	source $HOME/.zshrc
+}
+
 # ADD THE HOOK FUNCTIONS
 add-zsh-hook precmd  status
 add-zsh-hook preexec loggins
 #add-zsh-hook preexec explain
+add-zsh-hook zshexit shutdown
```

Comment:
I think we don't want this because we want the db to keep running between sessions. Wouldn't this mean if you ended one shell, the db would go down for the other? The db is kindof a singleton - a shared service.